modules/op-release-notes-1-8.adoc

```diff
@@ -20,6 +20,44 @@ With this update, you can run {pipelines-title} GA 1.8 and later on an OpenShift
 
 include::snippets/technology-preview.adoc[]
 
+=== Adding Git repositories by using webhooks
+
+With this update, as a developer, you can add your Git repository to {pac} by using the webhook mechanism. This way, you can use {pac} even when using GitHub Apps is not feasible.
+// (link:https://issues.redhat.com//browse/SRVKP-2137[SRVKP-2137 Pipeline as code - CLI add repository via webhooks])
+// Shivam Mukhade
+
+=== Running multiple PipelineRun objects
+
+Previously, as a developer, you could run only one `PipelineRun` object from your `.tekton` directory. With this update, you can run all the `PipelineRun` resources in your `.tekton` directory that match specified events by using the *Developer* perspective of the {product-title} web console. The console displays the status and reports of the runs. If you are using WebVCS, the reports display comments.
+// (link:https://issues.redhat.com//browse/SRVKP-1781[SRVKP-1781 Multiple Pipelines runs match on events])
+// Chmouel Boudjnah
+
+=== Installing Tekton Hub with better defaults
+
+With this update, as an administrator, you can install a local Tekton Hub instance with minimal input because it has better default configuration settings. You can start running your Tekton Hub instance and work with your teams to decide what customizations they need.
+// (link:https://issues.redhat.com//browse/SRVKP-2250[SRVKP-2250 Install Hub with default configs])
+// Shiv Verma
+
+=== Getting notifications if you add a duplicate repository
+
+With this update, as a developer, you get a notification from the `tkn-pac` CLI tool if you try to add a duplicate repository to a {pac} run. When you use`tkn pac repo create`, each repository must have a unique URL. This notification also helps prevent hijacking executions.
```

Comment:
```suggestion
With this update, as a developer, you get a notification from the `tkn-pac` CLI tool if you try to add a duplicate repository to a {pac} run. When you use`tkn pac repo create`, each repository must have a unique URL. This notification also helps prevent hijacking executions.
```
Is there a better phrase we could use here other than `hijacking executions`? That sounds just awful, imo.

Reply:
Updated. Thanks.